cpp/src/value_classes/ValueList.cpp

```diff
@@ -355,12 +355,12 @@ bool ValueList::GetItemLabels
 // Get the Item at the Currently selected Index
 //-----------------------------------------------------------------------------
 ValueList::Item const *ValueList::GetItem() const {
-	try {
-		return &m_items.at(m_valueIdx);
-	} catch (const std::out_of_range& oor) {
+	if ((m_valueIdx < 0) || (m_valueIdx >= (int32)m_items.size()))
```

Comment:
I had situations where m_ValueIdx was not assigned, and it had a value of -82385624
This case it checks also this